developer/customization/articles/02-architecture/03-understanding-services/02-osgi-cdi-integration/01-publishing-cdi-beans-as-osgi-services.markdown

```diff
@@ -0,0 +1,79 @@
+# Publishing CDI Beans as OSGi Services 
+
+OSGi CDI Integration lets you publish CDI beans as OSGi services, making them
+accessible via the Liferay's OSGi service registry. Here's how to publish your
+bean as a service:
+
+1.  Add a project dependency on the OSGi CDI Integration artifact. For example, 
+here's the dependency to use in a Maven `pom.xml` file:
+
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.service.cdi</artifactId>
```

Comment:
should you add `<version>1.0.0</version>` to make this snippet copy & paste ready?

Reply:
Added version. Thanks